core/src/test/java/org/apache/iceberg/TestMetrics.java

```diff
@@ -270,6 +289,97 @@ public void testMetricsForNullColumns() throws IOException {
     assertBounds(1, IntegerType.get(), null, null, metrics);
   }
 
+  @Test
+  public void testMetricsForTopLevelWithMultipleRG() throws Exception {
```

Comment:
"RG" should be expanded to `RowGroup`. We don't want to abbreviate unless the meaning is clear.

Reply:
updated.

Reply:
thanks @rdblue. please let me know if with changes it looks good.